unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesCardsConfigurator.cs

```diff
@@ -17,14 +17,16 @@ public static class PlacesCardsConfigurator
     /// <param name="placeInfo">Model to apply.</param>
     /// <param name="OnPlaceInfoClicked">Action to inform when the Info button has been clicked.</param>
     /// <param name="OnPlaceJumpInClicked">Action to inform when the JumpIn button has been clicked.</param>
-    public static PlaceCardComponentView Configure(PlaceCardComponentView placeCard, PlaceCardComponentModel placeInfo, Action<PlaceCardComponentModel> OnPlaceInfoClicked, Action<HotSceneInfo> OnPlaceJumpInClicked)
+    public static PlaceCardComponentView Configure(PlaceCardComponentView placeCard, PlaceCardComponentModel placeInfo, Action<PlaceCardComponentModel> OnPlaceInfoClicked, Action<HotSceneInfo> OnPlaceJumpInClicked, Action<string, bool> OnFavoriteClicked)
     {
         placeCard.Configure(placeInfo);
 
         placeCard.onInfoClick.RemoveAllListeners();
         placeCard.onInfoClick.AddListener(() => OnPlaceInfoClicked?.Invoke(placeInfo));
         placeCard.onJumpInClick.RemoveAllListeners();
         placeCard.onJumpInClick.AddListener(() => OnPlaceJumpInClicked?.Invoke(placeInfo.hotSceneInfo));
+        placeCard.onFavoriteClick -= OnFavoriteClicked;
+        placeCard.onFavoriteClick += OnFavoriteClicked;
```

Comment:
why do we need this magic? feels like we loose control over this object life-cycle

Reply:
As we are doing in the lines before where we remove listeners, we are in a scenario where the configurator is used by a  factory that gets the places cards from a pool

Reply:
This is a common pattern to avoid duplicates in listeners.

Reply:
ok